wg-naming/README.md

```diff
@@ -0,0 +1,70 @@
+<!---
+This is an autogenerated file!
+
+Please do not edit this file directly, but instead make changes to the
+sigs.yaml file in the project root.
+
+To understand how this file is generated, see https://git.k8s.io/community/generator/README.md
+--->
+# Naming Working Group
+
+
+## Stakeholder SIGs
+* SIG Architecture
+* SIG Contributor Experience
+* SIG Docs
+
+
+
+## Organizers
+
+* Celeste Horgan (**[@celestehorgan](https://github.com/celestehorgan)**), CNCF
+* Stephen Augustus (**[@justaugustus](https://github.com/justaugustus)**), VMware
+* Zach Corleissen (**[@zacharysarah](https://github.com/zacharysarah)**), Linux Foundation
+
+## Contact
+- Slack: [#wg-naming](https://kubernetes.slack.com/messages/wg-naming)
+- [Mailing list](https://groups.google.com/forum/#!forum/kubernetes-wg-naming)
+- [Open Community Issues/PRs](https://github.com/kubernetes/community/labels/wg%2Fnaming)
+<!-- BEGIN CUSTOM CONTENT -->
+
+**The following section will be reworked and formalized as a charter once the
+Working Group has been approved by the Steering Committee.**
+
+### Goals
+
+- Evaluate language and naming choices within the Kubernetes organization, with
+  a specific initial focus of:
+  - Removing language which cannot be fully separated from racist, sexist or
+    homophobic connotations in cultural contexts outside of computer
+    programming
+  - Improving clarity of codebases and documentation by replacing idioms,
+    metaphors and slang specific to the English language
+- Create a list of terms to avoid and propose alternatives
```

Comment:
Could we re-order this point with respect to creating the process and then provide clarity about whether this initial list would follow the same process that will be used for future terms?

Reply:
I'm not sure what you're looking for in regards to reordering the list. 

1. What do you think needs reordering?
2. What does reordering the list accomplish?

Reply:
> I'm not sure what you're looking for in regards to reordering the list.
> 
> 1. What do you think needs reordering?
> 2. What does reordering the list accomplish?

When reading this doc, I'm unsure whether 
 1. the process created to evaluate problematic language will be used to create this list of terms, or 
 2. whether some other process will be used to create the initial list since you already have a document started.

I think it would be a good idea to use the process proposed by this working group to drive the selection of terms for this list, but I'd like to see an explicit statement of intent either way.